### ISSUE TITLE
Dockerize document conversion utilities (pdftotext, antiword, docx2txt, wpd2html, tesseract, gs)

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#941]
This is coming out of conversations in #937 and in #936, in which @willglynn suggested/recommended pulling weird binaries into their own docker service or tool. Seems like a great suggestion. Let's do that.

**Comment by mlissner** [comment]
I still like this idea, but I must point out that:

1. https://github.com/minyk/open-ocr/ exists to provide a dockerized rest API to Tesseract. 

1. This is all now set up and working (I think, see https://github.com/freelawproject/courtlistener-task-server/issues/1), so I'm less inclined to care about this.

1. If we want to pull out the code for doing this kind of thing, it's actually probably pretty easy, just look at the commits around today and the dependencies in the task-server docker file.

**Comment by flooie** [comment]
We took a first step to dockerize a new functionality that handles tesseract and image processing. 
It's our first container running python 3.7 and it handles the extraction of data from financial disclosure documents. 

See the following repos for more information
[disclosure-extractor](https://github.com/freelawproject/disclosure-extractor)
[courtlistener-disclosure-extractor](https://github.com/freelawproject/courtlistener-disclosure-extractor)

To make it work nicely between containers - we built a flask micro service api endpoint to pass data back and forth between containers.

Also, we added a post_push hook and automatic building from commits to master to build and release new versions.
